### DISCUSSION TITLE
Automatic downscale of resolution after editing

### DISCUSSION THREAD

**Discussion by Chris2137** [hm21/pro_image_editor#52] [Q&A] [answered]
Hi @hm21 , 

  I used an image with original dimension : 3024 X 4032, after editing (by drawing using the painting mode) and save, it becomes 804 X 1072. 

  Is it configurable? I would like to keep the original resolution.

  Thanks.

**Comment by hm21** [comment]
Sounds to me like a bug where the pixel ratio is no longer correct. Can you post the `ProImageEditor` widget for me here so I can take a look on it how you configure it?

**Reply by Chris2137** [reply]
Thanks @hm21 . Here you go, I am on 2.7.9, made minor change on the doneEditing to just return the bytes instead of relying on onImageEditingComplete method :
              ProImageEditor.memory(
                editedImageByte!,
                key: _imageKey,
                allowCompleteWithEmptyEditing: true,
                configs: const ProImageEditorConfigs(
                  hideDoneButton: true,
                  removeTransparentAreas: true,
                  filterEditorConfigs: FilterEditorConfigs(
                    enabled: false,
                  ),
                  emojiEditorConfigs: EmojiEditorConfigs(
                    enabled: false,
                  ),
                  blurEditorConfigs: BlurEditorConfigs(
                    enabled: true,
                    maxBlur: 8,
                  ),
                ),
                onImageEditingComplete: (bytes) async {
                  //unused for now
                },
              ),

**Reply by hm21** [reply]
Okay, looks correct. On which platform (MacOS, Windows, Web, Linux...) do you have this problem? And the image you are inserting, what content-type does it have (for example "image/png")?

**Reply by Chris2137** [reply, marked as answer]
oh...I just realised the removeTransparentAreas is affecting the final output image. I just configure that because I found the final image is smaller.
I can get my original resolution after changing the removeTransparentAreas back to false. But the variable name is a bit confusing to me about the actual meaning.
Thanks anyway.
